cds_migrator_kit/rdm/streams.yaml

```diff
@@ -0,0 +1,14 @@
+db_uri: postgresql://cds-rdm:cds-rdm@localhost:5432/cds-rdm
+records:
+  thesis:
+    data_dir: cds_migrator_kit/rdm/data/thesis
+    tmp_dir: cds_migrator_kit/rdm/tmp/thesis
+    log_dir: cds_migrator_kit/rdm/log/thesis
+    extract:
+      dirpath: cds_migrator_kit/rdm/data/thesis/dump/
+    transform:
+      files_dump_dir: cds_migrator_kit/rdm/data/thesis/files/
+      missing_users: cds_migrator_kit/rdm/data/users
+      community_id: 96bf2d41-1f24-4025-baf7-65fd97c45aed
```

Comment:
I guess this is a test id for the moment?
It will required the larger discussion on how to organize content and communities with the library

Reply:
this is a local id for migration. In prod we have mounted a dedicated config (see [migration pod](https://paas.cern.ch/k8s/ns/cds-rdm-prod/configmaps/migration-stream-config))